<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>338</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>338</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>868</width>
    <height>338</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QLabel" name="subject_text_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ПРЕДМЕТ:</string>
   </property>
  </widget>
  <widget class="QLabel" name="homework_subject_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>30</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="notification_checkbox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Напоминать?</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendar">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>331</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="gridVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>20</width>
     <height>321</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="homework_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>271</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ADD_btn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ДОБАВИТЬ ДАТУ</string>
   </property>
  </widget>
  <widget class="QLabel" name="Datas_label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>50</y>
     <width>171</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>16</width>
     <height>321</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="time_text_label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>СПИСОК ДАТ ДЛЯ НАПОМИНАНИЯ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="REMOVE_btn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>УДАЛИТЬ ДАТУ</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_text_label2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>260</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ВРЕМЯ НАПОМИНАНИЯ</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="Time_hour_minutes">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>290</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>18</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="time">
    <time>
     <hour>18</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>30</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="status_label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>320</y>
     <width>981</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>-100</x>
     <y>0</y>
     <width>1241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
